src/main/java/org/javarosa/xpath/expr/XPathJoinChunkFunc.java

```diff
@@ -0,0 +1,66 @@
+package org.javarosa.xpath.expr;
+
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.DataInstance;
+import org.javarosa.xpath.XPathArityException;
+import org.javarosa.xpath.XPathNodeset;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+public class XPathJoinChunkFunc extends XPathFuncExpr {
+    public static final String NAME = "join-chunked";
+    // one or more arguments
+    private static final int EXPECTED_ARG_COUNT = -1;
```

Comment:
ah now I see that it's because this class overrides `validateArgCount()`. Could you update the comment above this line to just explain that? I think the current one is stale